Makefile

```diff
@@ -45,9 +45,9 @@ build: Dockerfile build/journalbeat
 build/journalbeat:
 	mkdir -p build
 	docker build -t $(IMAGE_NAME)-build .
-	docker run --name $(IMAGE_NAME)-build $(IMAGE_NAME)-build
-	-docker cp $(IMAGE_NAME)-build:/go/src/github.com/mheese/journalbeat/journalbeat build/journalbeat
-	docker rm $(IMAGE_NAME)-build
+	docker run --name journalbeat-build-container $(IMAGE_NAME)-build
```

Comment:
I was able to reproduce and this does resolve the issue. Please, however, use a variable called 'IMAGE_BUILD_NAME' at the top instead of hardcoding the container name all over the place. Other than that, looks good!